data/recipes/gcp_logging_collect.json

```diff
@@ -14,11 +14,15 @@
         "name": "GCPLogsCollector",
         "args": {
             "project_name": "@project_name",
-            "filter_expression": "@filter_expression"
+            "filter_expression": "@filter_expression",
+            "backoff": "@backoff",
+            "delay": "@delay"
         }
     }],
     "args": [
         ["project_name", "Name of the GCP project to collect logs from.", null],
-        ["filter_expression", "Filter expression to use to query GCP logs. See https://cloud.google.com/logging/docs/view/query-library for examples.", "resource.type = 'gce_instance'"]
+        ["filter_expression", "Filter expression to use to query GCP logs. See https://cloud.google.com/logging/docs/view/query-library for examples.", "resource.type = 'gce_instance'"],
+        ["--backoff", "If API query limits are exceeded, retry with an increased delay between each query to try complete the query at a slower rate.", false],
+        ["--delay", "Number of seconds to wait between each query to avoid hitting API query limits", 0]
```

Comment:
As per my other comment, you'll just need to copy these additions into the other recipes you've changed.